app/routes/items/detail.js

```diff
@@ -25,6 +26,11 @@ export default AuthorizeRoute.extend({
       await Ember.RSVP.all(promises);
     }
 
+    let detail_type = model.get("detailType");
```

Comment:
Could you use camel case for the variables, we're all having a hard time transitioning between ruby and javascript 🤣 
I'm very guilty of that as well, (especially due to my hatred of camel case haha)